config/locales/application_form.yml

```
@@ -818,6 +818,33 @@ en:
               in_future: Enter a year before %{date}
             other_grade:
               blank: Enter your grade
+        candidate_interface/science_gcse_grade_form:
+          attributes:
+            single_award_grade:
+              blank: Enter your science single award grade
+              invalid: Enter a real science single award grade
+            double_award_grade:
+              blank: Enter your science double award grade
+              invalid: Enter a real science double award grade
```

Don’t think we need say science here as already saying science in page title and elsewhere, so clear what this relates to.

```suggestion
            single_award_grade:
              blank: Enter your single award grade
              invalid: Enter a real single award grade
            double_award_grade:
              blank: Enter your double award grade
              invalid: Enter a real double award grade
```